full-stack-tests/core/webpack.config.js

```diff
@@ -47,6 +47,10 @@ function createConfig(shouldInstrument) {
         path: require.resolve("path-browserify"),
         stream: require.resolve("stream-browserify"),
         zlib: require.resolve("browserify-zlib"),
+        tty: require.resolve("tty-browserify"),
```

Comment:
where are these additions coming from?

Reply:
[error.txt](https://github.com/user-attachments/files/17138427/error.txt)
See the above for the failures in particular, I saw these errors pop up when I picked up your old PR, I assumed it was related to changing the typings, but after reverting the typings changes it still stuck around...

Reply:
well looking at the errors, prob coming from a dependency update, which is fine...